plugins/cue_points/quiz/lib/api/filters/KalturaQuizUserEntryFilter.php

```diff
@@ -5,4 +5,28 @@
  */
 class KalturaQuizUserEntryFilter extends KalturaQuizUserEntryBaseFilter
 {
+	/**
+	 * @var KalturaNullableBoolean
+	 */
+	public $isAnonymous;
+
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+		if(!is_null($this->isAnonymous))
+		{
+			if(KalturaNullableBoolean::toBoolean($this->isAnonymous)===false)
+			{
+				if($this->userIdNotIn==null)
+				{
+					$anonKuserIds = "";
+					$anonKusers = kuserPeer::getKuserByPartnerAndUids(kCurrentContext::getCurrentPartnerId(), array('', 0));
+					foreach ($anonKusers as $anonKuser) {
+						$anonKuserIds .= $anonKuser->getKuserId().",";
```

Comment:
@MosheMaorKaltura, since this is an API side filter, getPuserId() should be used here.
actually i think there's no need to look for these users and iterate each one - just put their names on userIdNotIn.